operate/qa/integration-tests/src/test/java/io/camunda/operate/schema/ImportStoreIT.java

```diff
@@ -65,8 +64,6 @@ public class ImportStoreIT extends AbstractSchemaIT {
 
   @Autowired public SchemaManager schemaManager;
 
-  @Autowired public ImportPositionIndex importPositionIndex;
```

Comment:
❓ same here?
if we added the beans to the correct configuration classes, those wouldn't have needed to be touched.

Some of those tests should be removed or migrated to exporter at some point anyway, but we didn't need to change them as part of this PR, I guess